### DISCUSSION TITLE
UNIQUE constraint

### DISCUSSION THREAD

**Discussion by Zer0xxxx** [alfateam/rdb#98] [Ideas]
Hello!
First of all, thank you a lot for this fp ORM.

I would like to know if it is planned to add a `.unique()` composite function in Orange ORM. 

Like:
```sql
related_xxx_id BIGINT UNIQUE NOT NULL,
```

**Comment by lroal** [comment]
Adding constraints to the sql only makes sense if Orange can create sql migrations. And it does not per se.
However, I am considering adding a `sql` cli command for generating the entire db schema. When that happens, it also makes sense adding a unique() constraint.

**Reply by Zer0xxxx** [reply]
It would be fantastic to generate the DB schema from the Orange map!
